README.md

```diff
@@ -74,31 +99,73 @@ __NOTE__: You can use [terraform-aws-organization-access-group](https://github.c
-| `master_account_id`     | ``                                | The ID of the master account to grant permissions to access the current account                    | Yes      |
-| `role_name`             | `OrganizationAccountAccessRole`   | The name of the role to grant permissions to delegated IAM users in the master account to the current account   | No      |
-| `policy_arn`            | `arn:aws:iam::aws:policy/AdministratorAccess`  | Policy ARN to attach to the role. By default it attaches `AdministratorAccess` managed policy to grant full access to AWS services and resources in the current account   | No      |
 
 
+
+## Makefile Targets
+```
+Available targets:
+
+  help                                This help screen
+  help/all                            Display help for all targets
+  lint                                Lint terraform code
+
+```
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| master_account_id | The ID of the master account to grant permissions to access the current account | string | - | yes |
+| policy_arn | Policy ARN to attach to the role. By default it attaches `AdministratorAccess` managed policy to grant full access to AWS services and resources in the current account | string | `arn:aws:iam::aws:policy/AdministratorAccess` | no |
+| role_name | The name of the role to grant permissions to delegated IAM users in the master account to the current account | string | `OrganizationAccountAccessRole` | no |
+
 ## Outputs
 
-| Name                | Description                                         |
-|:--------------------|:----------------------------------------------------|
-| `role_name`         | The name of the crated role                         |
-| `role_id`           | The stable and unique string identifying the role   |
-| `role_arn`          | The Amazon Resource Name (ARN) specifying the role  |
+| Name | Description |
+|------|-------------|
+| role_arn | The Amazon Resource Name (ARN) specifying the role |
+| role_id | The stable and unique string identifying the role |
+| role_name | The name of the crated role |
+
+
+
+
+## Related Projects
+
+Check out these related projects.
+
+- [terraform-aws-organization-access-group](https://github.com/cloudposse/terraform-aws-organization-access-group) - Terraform module to create an IAM Group and Policy to grant permissions to delegated IAM users in the Organization's master account to access a member account
 
 
 
 ## Help
 
 **Got a question?**
 
-File a GitHub [issue](https://github.com/cloudposse/terraform-aws-organization-access-role/issues), send us an [email](mailto:hello@cloudposse.com) or reach out to us on [Gitter](https://gitter.im/cloudposse/).
+File a GitHub [issue](https://github.com/cloudposse/terraform-aws-organization-access-role/issues), send us an [email][email] or join our [Slack Community][slack].
+
+## Commerical Support
```

Comment:
@vadim-hleif this typo will need to be fixed by regenerating the README - since this is now merged: https://github.com/cloudposse/build-harness/commit/17b052bb53da80b80acbf3b7dc3c5e32bbfeb16b